pyrk/convective_model.py

```diff
@@ -0,0 +1,81 @@
+from utilities.ur import units
+from materials.material import Material
+
+
+class ConvectiveModel(object):
```

Comment:
This is great. I very much like how it's been implemented. I salute you. That said, it might have been better to make this it's own PR (to keep it separate from the unrelated multipoint model PR). That's not a big deal today, but is perhaps something to consider when adding features in the future. 

Also this class has no tests. It needs and deserves a few tests. Please add a set of tests for this class that is capable of demonstrating its use, its validity, and its correct calculation of known convection coefficients.